src/js/TreeView.js

```diff
@@ -138,7 +141,36 @@ TreeView.appItem = function(label, node) {
 	var item = new TreeViewItem(label);
 	item.node = node;
 	item.element.className = 'ngi-app';
-	return item;
+
+	// Highlight DOM elements the scope is attached to when hovering the item
+	// in the inspector
+	item.element.addEventListener('mouseover', function(event) {
+		if(event.target.nodeName === 'LABEL' && event.target.parentNode.classList.contains('ngi-scope')) {
+			// Do not add a layer when mouse comes from ngi-annotation
+			if (event.relatedTarget.classList.contains('ngi-annotation')) return false;
```

Comment:
Curious: Why the change from checking for the "caret" to checking for the annotation?